src/shared/Progression.h

```diff
@@ -27,8 +27,10 @@
 #define CLIENT_BUILD_1_8_4 4878
 #define CLIENT_BUILD_1_9_4 5086
 #define CLIENT_BUILD_1_10_2 5302
+#define CLIENT_BUILD_1_11_0 5428
 #define CLIENT_BUILD_1_11_2 5464
 #define CLIENT_BUILD_1_12_1 5875
+#define CLIENT_BUILD_2_0_0 5610
```

Comment:
Don't add TBC builds, again somebody might try to set SUPPORTED_CLIENT_BUILD to it, and we do not support TBC.

Reply:
There are code to 2.0 clients to logon. As I said before, it is just to not use "magic numbers" inside code and to identify it.

Reply:
I think `CLIENT_BUILD_2_0_0 ` should be defined as the actual TBC Build number, but not available in the make so people don't get the wrong idea that it actually is supported. All custom versions should be between `CLIENT_BUILD_1_12_1 = 5875` and `CLIENT_BUILD_2_0_0 = ???? `.

Reply:
I will fix CLIENT_BUILD_2_0_0 to right one: 6080. I put an alpha version as mistake.

Reply:
My reference: https://archive.org/download/World_of_Warcraft_Client_and_Installation_Archive/ISO/

Reply:
I deleted Progression.h entries. I didn't know it is generated for cmake and it is in .gitignore.
Now my code passes on github's checks.